### PR TITLE
Create generic/shared component for Metrics 

### PR DESCRIPTION
**Pull request by Msarawan**
Memory and CPU Charts:

Screenshot from 2023-06-20 20-47-45

### CODE REVIEW

File: src/brokers/broker-details/components/Topics/Topics.component.tsx [hunk 1]
```diff
@@ -28,24 +28,26 @@ const Topics: React.FC<TopicsProps> = ({ topicData, isLoaded, loadError }) => {
...
-          <TopicsRow
-            obj={obj}
-            rowData={rowData}
-            activeColumnIDs={activeColumnIDs}
-            columns={columns}
-          />
-        )}
-      />
+      </div>
+      <div>
```
Comment (ajaypratap003) [inline src/brokers/broker-details/components/Topics/Topics.component.tsx:34]: Do we need this extra `div` ?

File: src/brokers/metrics/components/CardBrokerCPUUsageMetrics/CardBrokerCPUUsageMetrics.container.tsx [hunk 2]
```diff
@@ -63,6 +72,12 @@ export const CardBrokerCPUUsageMetricsContainer: FC<
       fixedXDomain={xDomain}
       samples={samples}
       formatSeriesTitle={(labels) => labels.pod}
+      title={t('cpu_usage')}
+      helperText={t('cpu_usage_help_text')}
+      dataTestId={'metrics-broker-cpu-usage'}
+      yTickFormat={yTickFormat}
+      ariaTitle={ariaTitle}
```
Comment (ajaypratap003) [inline src/brokers/metrics/components/CardBrokerCPUUsageMetrics/CardBrokerCPUUsageMetrics.container.tsx:79]:
```suggestion
      ariaTitle={t('cpu_usage')}
```

File: src/brokers/metrics/components/CardBrokerMemoryUsageMetrics/CardBrokerMemoryUsageMetrics.container.tsx [hunk 3]
```diff
@@ -68,6 +91,14 @@ export const CardBrokerMemoryUsageMetricsContainer: FC<
       fixedXDomain={xDomain}
       samples={samples}
       formatSeriesTitle={(labels) => labels.pod}
+      title={t('memory_usage')}
+      helperText={t('memory_usage_help_text')}
+      dataTestId={'metrics-broker-memory-usage'}
+      yTickFormat={yTickFormat}
+      processedData={processedData}
+      // data={data}
+      label={label}
```
Comment (ajaypratap003) [inline src/brokers/metrics/components/CardBrokerMemoryUsageMetrics/CardBrokerMemoryUsageMetrics.container.tsx:100]:
```suggestion
      label={'\n\n\n\n' + t('axis_label_bytes')}
```

File: src/shared-components/QueryBrowser/QueryBrowser.tsx [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+                <ChartAxis
+                  label={label}
+                  crossAxis={false}
+                  tickCount={6}
+                  dependentAxis
+                  tickFormat={yTickFormat}
+                />
+                <ChartGroup>
+                  {metricsType === 'memory'
+                    ? processedData.map((values, index) => {
```
Comment (ajaypratap003) [inline src/shared-components/QueryBrowser/QueryBrowser.tsx:192]: Add condition for array only , rest code keep same.

[metricsType === 'memory' ?  processedData: data].map((values, index)=>{
  //add code here
});

By doing this we can avoid to write duplicate code.

**Review by ajaypratap003 — CHANGES_REQUESTED** [review]
Please follow inline review comments.

File: src/brokers/broker-details/components/Topics/Topics.component.tsx (hunk 1 above)
Comment (Msarawan) [reply src/brokers/broker-details/components/Topics/Topics.component.tsx:34]: > Do we need this extra div ? 
->  not necessary i was just checking by adding this does it stop crashing table alignment issue. i will remove it.

**Review by ajaypratap003 — DISMISSED** [review]
As after removing `PageSection` tag, table alignment issue doesn't solve. 
Can you undo `PageSection` changes . Rest looks good.

**Review by ajaypratap003 — APPROVED** [review]

**Pull request merged**